yucca/preprocessing/YuccaPreprocessor.py

```diff
@@ -77,7 +77,7 @@ def initialize_paths(self):
         self.input_dir = join(yucca_raw_data, self.task)
         self.imagepaths = subfiles(join(self.input_dir, "imagesTr"), suffix=self.image_extension)
         self.imagepaths = subfiles(join(self.input_dir, "imagesTr"), suffix=self.image_extension)
-        self.subject_ids = subfiles(join(self.input_dir, "labelsTr"), suffix=self.image_extension, join=False)
+        self.subject_ids = subfiles(join(self.input_dir, "labelsTr"), join=False)
```

Comment:
Yes this is definitely a mistake -- sorry!  But we have to at least exclude files that start with a `.` eg. `.DS_STORE`. I would recommend figuring out the `self.label_extension` and using that as the suffix.